build/_headers

```diff
@@ -1,4 +1,5 @@
 /*
   Access-Control-Allow-Origin: *
+  SameSite: None; Secure
```

Comment:
```suggestion
  Set-Cookie: SameSite=None
  Set-Cookie: Secure
```
As far as I can see `SameSite` is a `Set-Cookie` attribute, not an independent header.

Reply:
Though, I'm not 100% sure, since `Set-Cookie` should have the following form

```
Set-Cookie: <cookie-name>=<cookie-value>; SameSite=None
```

I'll take a closer look.

Reply:
> As far as I can see `SameSite` is a `Set-Cookie` attribute, not an independent header.

You are correct, my code suggestion was only an indicator of how it could be done. Looks like @Twaha-Rahman merged it as is. I should have clarified that it needs to be validated and confirmed. Apologies.

Reply:
My apologies too..... looks like I overlooked a lot of things.... I am looking at the specifics right now!

Reply:
According to MDN, we would normally need to have something like this -> 

`Set-Cookie: <cookie-name>=<cookie-value>; SameSite=None; Secure;`

But for the Netlify version of things, which is used by FCC's CDN will look like this ->

`Set-Cookie = "<cookie-name>=<cookie-value>; SameSite=None; Secure;"`

(Btw @raisedadead I couldn't get the Netlifly's Header Playground to work)

Reply:
@Twaha-Rahman, don't worry, we'll get there.  

For the playground, just drop the text of `_headers` in there, click 'Test rules' and you should see if it's working at the bottom.

Reply:
@ojeytonwilliams Okay... got the playground working. Thanks!